From: Thorsten Behrens <tbehrens@suse.com>
Date: Tue, 5 Aug 2014 10:01:42 +0200
Subject: [PATCH] Fix bnc#890345 - wrong service name for logrotate.

We've renamed the service to ceph-radosgw, reflect that in
logrotate config, too.
---
 src/rgw/logrotate.conf | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/rgw/logrotate.conf b/src/rgw/logrotate.conf
index ec47f00..7e527e8 100644
--- a/src/rgw/logrotate.conf
+++ b/src/rgw/logrotate.conf
@@ -7,7 +7,7 @@
         if which invoke-rc.d > /dev/null 2>&1 && [ -x `which invoke-rc.d` ]; then
             invoke-rc.d radosgw reload >/dev/null
         elif which service > /dev/null 2>&1 && [ -x `which service` ]; then
-            service radosgw reload >/dev/null
+            service ceph-radosgw reload >/dev/null
         fi
         # Possibly reload twice, but depending on ceph.conf the reload above may be a no-op
         if which initctl > /dev/null 2>&1 && [ -x `which initctl` ]; then
